File: 2244. Minimum Rounds to Complete All Tasks/map.py
class Solution:
    def minimumRounds(self, tasks: List[int]) -> int:
        task = Counter(tasks)

        if 1 in task.values():
            return -1

        round = 0
        for num in task.values():
            round += (num // 3) + bool(num % 3)

        return round

File: 2244. Minimum Rounds to Complete All Tasks/map.cpp
class Solution{
public:
    int minimumRounds(vector<int>& tasks){
        unordered_map<int, int> cnt;
        for (int i: tasks)
            cnt[i]++;

        int round = 0;
        for (auto &it: cnt){
            if (it.second == 0) return -1;
            round += (it.second + 2) / 3;
        }
        return round;
    }
};